### PR TITLE
React accordion component - Matthew Kweon

### PR DESCRIPTION
**Pull request by MattKweon**
Kapture 2022-11-01 at 14 58 25

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+}
+
+const accordionData = [
+  {
+    id: '1',
+    name: 'Hypertext Markup Language',
+    content: `Hypertext Markup Language (HTML) is the standard markup
+              language for creating web pages and wev applications. With
+              Cascading Style Sheets (CSS) and JavaScript, it forms a
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx]: This array should be passed to the component as a prop in `index.jsx`

File: react-accordion-component/src/accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  handleClick(e) {
+    const id = e.target.getAttribute('data-id');
+    this.setState({
+      onClick: id,
+      isOpen: true
+    });
+    if (id === this.state.onClick) {
+      this.setState({ isOpen: false });
+    }
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx]: Try modifying your solution to only need one property in state. If the user clicks the same topic again, set the state back to null

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Excellent, just some small tweaks

**Comment by MattKweon** [comment]
Screen Shot 2022-11-01 at 11 41 50 PM

**Review by thebearingedge — APPROVED** [review]
💯

**Pull request merged**